modules/installation-osp-kuryr-known-limitations.adoc

```diff
@@ -11,7 +11,10 @@ Using {product-title} with Kuryr SDN has several known limitations.
 [id="openstack-general-limitations_{context}"]
 == {rh-openstack} general limitations
 
-{product-title} with Kuryr SDN does not support NodePort services.
+{product-title} with Kuryr SDN does not support NodePort services. Also, if
```

Comment:
> Also, if the Machines Subnet is not connected to a Router or it’s connected, but the Router has no external gateway set, Kuryr won’t create Floating IPs for Services with LoadBalancer type.

-> 

> [line break]
> If the machines subnet is not connected to a router, or if the subnet is connected, but the router has no external gateway set, Kuryr cannot create floating IPs for load balancer services.

Reply:
Sounds good.

Reply:
I modified  "load balancer services" to "LoadBalancer services" to give a better indication that it's the type of the Service.

Reply:
>[line break]
>If the machines subnet is not connected to a router, or if the subnet is connected, but the router has no external gateway set, Kuryr cannot create floating IPs for LoadBalancer services.

That, then?

Reply:
Committing directly.